### ISSUE TITLE
r2pm -i radeco :: fails with r2pipe::structs::LVarRef missing method chars

### ISSUE THREAD

**Issue by smarek** [radare/radeco-lib#52]
```
error: no method named `chars` found for type `r2pipe::structs::LVarRef` in the current scope
   --> /home/smarek/.cargo/git/checkouts/radeco-lib-bf219eb1a7e916cc/ebc1b61b52408c5d4cc530298894936eb60f264e/src/frontend/containers.rs:285:26
    |
285 |         for c in ref_str.chars() {
    |                          ^^^^^

error: aborting due to previous error

Build failed, waiting for other jobs to finish...
error: Could not compile `radeco-lib`.
```

I assume this is known issue, but I report for tracking purposes

**Comment by sushant94** [comment]
Fixed in 59fb1d8aa7556bc15e4b69001c512dbf64685dd3

**Comment by radare** [comment]
😍
